src/index.ts

```diff
@@ -166,6 +168,7 @@ namespace Experimental {
   // indexed merkle map
   export let IndexedMerkleMap = Experimental_.IndexedMerkleMap;
   export type IndexedMerkleMap = IndexedMerkleMapBase;
+  export type IndexedMerkleMapBase = ExperimentalIndexedMerkleMapBase;
```

Comment:
We already expose `IndexedMerkleMapBase` (see the line above) via
```ts
export type IndexedMerkleMap = IndexedMerkleMapBase;
```

does this not work for you?

Reply:
> does this not work for you?

Yes, it works if I import `IndexedMerkleMap` two times - as the function and as the IndexedMerkleMapBase type:

```typescript
const { IndexedMerkleMap } = Experimental;
type IndexedMerkleMap = Experimental.IndexedMerkleMap;
```

Then, I use both the function and the type in the serializing code, which is a little bit confusing, as the same name is used for the type that contains the data that I use and the function, but it works and I can even rename the IndexedMerkleMap back to IndexedMerkleMapBase while importing:
```typescript
const { IndexedMerkleMap } = Experimental;
type IndexedMerkleMapBase = Experimental.IndexedMerkleMap;
```

Reply:
> I would be in favor of not exporting another type, just for the sake of simplicity. This will be fixed indirectly when we move `IndexedMerkleMap` out of the experimental namespace

I agree. Let's wait for a move of `IndexedMerkleMap` out of the experimental namespace. I hope it will be soon, as I plan to use it on the mainnet!